app/ts/components/Import.tsx

```diff
@@ -8,6 +7,7 @@ import { AppSettings, Bundle, serialize, Signers } from '../types/types.js'
 import { EthereumAddress } from '../types/ethereumTypes.js'
 import { TransactionList } from '../types/bouquetTypes.js'
 import { ImportModal } from './ImportModal.js'
+import { getAddress } from 'ethers'
```

Comment:
Super nitpicky style: I personally prefer sorting external imports above internal imports, like you had originally.  🤷